packages/react/src/themes/base/components/Slider/sliderVariables.ts

```diff
@@ -0,0 +1,48 @@
+import { pxToRem } from '../../../../lib'
+import { SiteVariablesPrepared } from '../../../types'
+
+export interface SliderVariables {
+  height: React.CSSProperties['height']
```

Comment:
Styles should be framework agnostic and variables, too. So we should not use `React.*` stuff

Reply:
done, thanks